eth_tester/backends/pyevm/main.py

```diff
@@ -465,9 +465,9 @@ def _max_available_gas(self):
     @replace_exceptions({
         EVMInvalidInstruction: TransactionFailed,
         EVMRevert: TransactionFailed})
-    def estimate_gas(self, transaction):
+    def estimate_gas(self, transaction, block_number="latest"):
         evm_transaction = self._get_normalized_and_unsigned_evm_transaction(assoc(
-            transaction, 'gas', 21000))
+            transaction, 'gas', 21000), block_number)
```

Comment:
*nitpick*


I find this visually harder to parse, and [this style section](https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#first-thing-on-its-own-line) implies that the preferred approach is:

```py
        evm_transaction = self._get_normalized_and_unsigned_evm_transaction(
            assoc(transaction, 'gas', 21000),
            block_number,
        )
```

It stands out to me more, now that there's >1 argument to the method.